webpack.config.js

```diff
@@ -24,15 +25,18 @@ module.exports = {
         path: path.resolve(__dirname, 'build'),
         filename: '[name].js'
     },
+    resolve: {
+        symlinks: false
+    },
     externals: {
         React: 'react',
         ReactDOM: 'react-dom'
     },
     module: {
         rules: [{
-            test: /\.jsx?$/,
+            // allow ES2015 in any *.js or *.jsx file under .../scratch-*/src/...
+            test: /[\\/]+scratch-[^\\/]+[\\/]+src[\\/]+.+\.jsx?$/,
```

Comment:
Any way we can make this more readable using more than one regex? It seems like we should be able to use the other tools available to `Condition` https://webpack.js.org/configuration/module/#condition